### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by christyjacob4**
## What does this PR do?
Update readme with new install commands

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [ ] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 English | [简体中文](README-CN.md)
 
-[**Announcing Appwrite 1.2 with GraphQL support! Learn what's new!**](https://appwrite.io/graphql)
+<!-- [**Announcing Appwrite 1.3 with Database Relations! Learn what's new!**](https://appwrite.io/graphql) -->
```
Comment (Meldiron) [inline README.md:29]: `Database Relations` or `Database Relationships`? Not sure whats official name.

[the remark this answers is not in the document]
Comment (christyjacob4) [reply README.md:29]: I have commented it. So we can replace it tmro

Comment (Meldiron) [reply README.md:29]: Let me know either way, I will double check articles

Comment (christyjacob4) [reply README.md:29]:
```suggestion
<!-- [**Announcing Appwrite 1.3 with Database Relationships! Learn what's new!**](https://appwrite.io/graphql) -->
```

[the remark this answers is not in the document]
Comment (Meldiron) [reply README.md:29]: Nevermind. I just noticed it's a comment.

**Pull request merged**